spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
```

Comment:
Yes, I was told how the let works and it was little wrong. So let! is evaluated repeatedly before each example? So it is slower and must be avoid if it is not necessary. Did a got it right?